.github/workflows/pr-labeler.yml

```diff
@@ -25,9 +25,9 @@ jobs:
           script: |
             const maintainers = [
               'aws-sam-cli-bot',
-              'jfuss', 'hoffa', 'awood45', 'aahung', 'hawflau', 'mndeveci', 'ssenchenko',
-              'qingchm', 'moelasmar', 'xazhao', 'mildaniel', 'marekaiv', 'torresxb1',
```

Comment:
Should we keep people that are still internal at Amazon? I wonder if we remove some of these people if we'll have large spikes in our responses dashboard if they were the last ones to respond to an issue

Reply:
I suppose we could still keep the internal people, but this GH action would be run on new issues and PR's right? This would not impact any previous issues or issues.

Reply:
That's right, these only run on new issues, good call.

Let's discuss who should stay on this list offline.